R/wrappers.R

```diff
@@ -16,3 +16,5 @@ isRealReallyInt = function(x) .Call(CisRealReallyIntR, x)
 isReallyReal = function(x) .Call(CisReallyReal, x)
 
 coerceAs = function(x, as, copy=TRUE) .Call(CcoerceAs, x, as, copy)
+
+topn = function(x, n, na.last=TRUE, decreasing=FALSE, sorted=FALSE) .Call(Ctopn, x, as.integer(n), na.last, decreasing, sorted)
```

Comment:
I do think this will be most useful when equipped to sort multiple vectors, so my sense is the signature should be `topn(n, x, ...)` or `topn(n, ...)`. 

Here's code that looks `topn()`-able:

https://github.com/search?q=lang%3AR+%2Forder%5C%28.*%5C%29%5C%5B1%3A%5B0-9%5D%2F&type=code

That said, I could only really find one usage that needs multiple vectors (could be I have a bad regex)!

https://github.com/search?q=lang%3AR+%2F%5B%5Ea-z0-9._%5Dorder%5C%28%5B%5E%28%29%3D%5D*%2C%5B%5E%3D%5Cn%5C%5B%5C%5D%29%5D*%5C%29%5C%5B1%3A%5B0-9%5D%2F&type=code

Reply:
But n will be most often scalar, no? Then it is not clear why it would be first argument. Then it is also not pipeable for data input.

Reply:
I like the idea of adding `...` support, however, being pipeable seems more advantageous nowadays.